green_cli/utils.py

```diff
@@ -0,0 +1,19 @@
+import click
+import json
+
+import greenaddress as gdk
+
+from green_cli.gdk_resolve import gdk_resolve
+
+def get_transaction(session, txid):
+    # TODO: Iterate all pages
+    # 900 is slightly arbitrary but currently the backend is limited to 30 pages of 30
```

Comment:
this is no longer true as of the last gdk release (the backend is still limited but gdk works around it)

Reply:
Paging would be more efficient here though, until gdk has a persistent tx cache

Reply:
Yes this should change but I will leave that as a separate issue as this PR has only moved this code